src/components/forward/forward.js

```diff
@@ -30,6 +30,21 @@ const mapStateToProps = state => ({
   isDvr: state.engine.isDvr,
   isLive: state.engine.isLive
 });
+
+/**
+ * translates
+ * @param {any} props - Props
+ * @returns {Object} - The object translations
+ */
+const translates = (props: any) => ({
+  forwardText: !props.step ? (
+    <Text id={'controls.forward'}>Seek forward</Text>
+  ) : (
+    <Text id={'controls.secondsForward'} fields={{seconds: props.step}}>
+      {`Seek ${props.step} seconds forward`}
```

Comment:
Why are you repeating the same string that you wrote in the json file ????   

It makes no sense and redundant 
See here : https://www.npmjs.com/package/preact-i18n#pluralization-and-templating